core/modules/ccontrol/UserQuerySelect.cc

```diff
@@ -248,11 +247,7 @@ QueryState UserQuerySelect::join() {
     bool successful = _executive->join(); // Wait for all data
```

Comment:
Will this return `false` if the query was only partially submitted (`_executive->getCancelled()` was true in the above loop)?